docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -365,7 +365,7 @@ Release
         export VERSION=0.1.2
 
```

Comment:
This creates two sections of code (the export and the task), with no intervening text. It looks strange.